data/json/monsters.json

```diff
@@ -411,6 +411,50 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_gas_zombie",
+    "type": "MONSTER",
+    "name": "gasoline zombie",
+    "description": "A huge bloated zombie that appears to have fed upon gasoline, fumes and flames escape from it's mouth and fuel leaks from it's waddling form.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 5,
+    "volume": "102500 ml",
+    "weight": 160000,
+    "hp": 90,
+    "speed": 25,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "dark_gray_red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 3,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 0,
+    "vision_night": 3,
+    "luminance": 8,
+    "harvest": "exempt",
+    "special_attacks": [ [ "scratch", 20 ] ],
+    "death_drops": "default_zombie_items",
+    "death_function": [ "CONFLAGRATION" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "DRIPS_GASOLINE",
+      "PUSH_MON",
+      "PATH_AVOID_DANGER_1",
```

Comment:
This isn't particularly zombish.

Reply:
The flags? they are the same as a boomer?

Reply:
I meant specifically `PATH_AVOID_DANGER_1`. I wonder is there a reason why boomers have it?

Reply:
No idea, that's weird, but this isn't really a boomer, so I can remove that to make it more in line with other zombies